base/strings/util.jl

```diff
@@ -224,6 +224,7 @@ end
     chopprefix(s::AbstractString, prefix::Union{AbstractString,Regex}) -> SubString
 
 Remove the prefix `prefix` from `s`. If `s` does not start with `prefix`, a string equal to `s` is returned.
+Note, for case-insensitive, e.g. for shortening regular URLs, use with a Regex.
```

Comment:
Something is off here. "Note, for case-insensitive, " is missing a word. Maybe you mean something like
```suggestion
For case-insensitive comparison of the prefix, e.g. for shortening regular URLs, you can use a suitable `Regex` for `prefix`.
```

But also: what is a "regular URL"?

Reply:
URLs actually allow file://, and historically ftp:// (all browsers have dropped that...), so shortening URLs, is practically ok, those exceptions will NOT be shortened (but that's also ok). That what I had in mind. I can go with dropping "regular" as confusing. [There are also URL shortening services, what I did NOT have in mind...]

Actually http and https CAN point to entirely different content (always bad), or only one or the other working, then most likely http, but browsers by now default to https if neither is given. Shortening should be ok in almost all cases, why I feel this is an ok example, though I'm not trying to promote it as such. However, if you WANT to do it it's non-trivial enough, that it can serve as an example.

Reply:
Yeah sure, sure, but this is going way off tangent. The example feels overly specific. Maybe this was the use case you had in mind, but if one has to first explain it at length, it is not good. I'll make an alternative suggestion